server/sqlstore/playbook.go

```diff
@@ -403,3 +455,12 @@ func directionOptionToSQL(direction playbook.SortDirection) string {
 		return ""
 	}
 }
+
+func correctPaginationOpts(opts *playbook.Options) {
+	if opts.PerPage <= 0 {
+		opts.PerPage = 1000
+	}
+	if opts.Page <= 0 {
```

Comment:
The nittiest nit you can imagine:
```suggestion
	if opts.Page < 0 {
```

Reply:
That's pretty nitty, but I'll own it. Fixed. 😂